config/jobs/kubernetes/sig-azure/sig-azure-config.yaml

```diff
@@ -0,0 +1,81 @@
+presets:
+- labels:
+    preset-azure-cred: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: REGISTRY
+    value: ritazh # update this after we have another account to use
+  - name: DOCKER_USERNAME
+    value: ritazh # update this after we have another account to use
+  - name: DOCKER_PASSWORD_FILE
+    value: /etc/docker-cred/dockercred
+  volumes:
+  - name: azure-cred
+    secret:
+      secretName: azure-cred
+  - name: azure-ssh
+    secret:
+      secretName: azure-ssh
+  - name: docker-cred
+    secret:
+      secretName: docker-cred
+  volumeMounts:
+  - name: azure-cred
+    mountPath: /etc/azure-cred
+    readOnly: true
+  - name: azure-ssh
+    mountPath: /etc/azure-ssh
+    readOnly: true
+  - name: docker-cred
+    mountPath: /etc/docker-cred
+    readOnly: true
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: ritazh/kubekins-e2e:v20190126-731395d-dirty-master # update this after test-infra PRs are merged 
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--root=/go/src"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--repo=k8s.io/kubernetes=$(PULL_REFS)"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=460"
+        - "--scenario=kubernetes_e2e"
+        - --
+        - "--gce-ssh=" # https://github.com/kubernetes/test-infra/pull/10054
+        - "--test=true"
+        - "--up=true"
+        - "--down=true"
+        - "--deployment=acsengine"
+        - "--build=bazel" # build kubectl binary
+        - "--provider=skeleton" # noop
+        - "--ginkgo-parallel=30"
+        - "--acsengine-agentpoolcount=3"
```

Comment:
noooope, I do not want us blocking merges on a non-opensource kubernetes

Reply:
And the tool itself is open source as well - https://github.com/azure/aks-engine

Reply:
the PR has been updated to `always_run: false`, it will not block merges. resolving.